### PR TITLE
Bug fix for Gnav Multi Sign-in

### PR DESCRIPTION
**Pull request by rgclayton**
Attempting to sign-in to the `Adobe Account` from the multi sign-in dropdown was not correctly logging in to IMS.

**Bacom Test URLs:**
- Before: https://main--bacom--adobecom.hlx.page/drafts/rclayton/multi-cloud-signin?martech=off
- After: https://main--bacom--adobecom.hlx.page/drafts/rclayton/multi-cloud-signin?milolibs=rclayton-multi-nav-signin&martech=off

**Milo Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/drafts/rclayton/multi-cloud-signin?martech=off
- After: https://rclayton-multi-nav-signin--milo--adobecom.hlx.page/drafts/rclayton/multi-cloud-signin?martech=off

### CODE REVIEW

**Comment by rgclayton** [comment]
@auniverseaway Is this the correct experience when coming from Milo?

**Review by chrischrischris — APPROVED** [review]

File: libs/blocks/gnav/gnav.js [hunk 1]
```diff
@@ -423,29 +423,29 @@ class Gnav {
...
   decorateSignIn = (blockEl, profileEl) => {
-    const profileDropDown = blockEl.querySelector(':scope > div:nth-child(2)');
+    const dropDown = blockEl.querySelector(':scope > div:nth-child(2)');
     const signIn = blockEl.querySelector('a');
 
     signIn.classList.add('gnav-signin');
     signIn.setAttribute('daa-ll', 'Sign In');
 
-    if (profileDropDown) {
+    const signInEl = dropDown ? dropDown.querySelector('li:last-of-type a') : profileEl;
```
Comment (meganthecoder) [inline libs/blocks/gnav/gnav.js:432]: super nit:
```suggestion
    const signInEl = dropDown?.querySelector('li:last-of-type a') || profileEl;
```

**Review by meganthecoder — APPROVED** [review]

**Comment by auniverseaway** [comment]
Looks like we're still getting the extra spacing from `.has-menu`.

Screen Shot 2022-10-28 at 8 32 34 AM

**Comment by auniverseaway** [comment]
> I left this on purpose. I noticed that removing it reintroduced the CLS issue.
CLS after login won't be avoidable to have it be the right size. Keep in mind: Google never logs in, you won't get dinged for this CLS.

**Comment by rgclayton** [comment]
> > I left this on purpose. I noticed that removing it reintroduced the CLS issue.
> 
> CLS after login won't be avoidable to have it be the right size. Keep in mind: Google never logs in, you won't get dinged for this CLS.

Gotcha. Thanks! Pushed an update. The spacing issue should be fixed now.

**Pull request merged**